examples/with-mdx-remote/utils/mdxUtils.js

```diff
@@ -0,0 +1,11 @@
+import fs from 'fs'
+import path from 'path'
+
+// POSTS_PATH is useful when you want to get the path to a specific file
+export const POSTS_PATH = path.join(process.cwd(), process.env.mdxPath)
```

Comment:
Can we switch to use `posts` here instead of `process.env.mdxPath`? That way we can remove `next.config.js`